src/modules/navigation/tree/TreeView.tsx

```diff
@@ -100,6 +100,13 @@ const TreeView = ({
     ? getIdsFromPath(focusedItem.path)
     : defaultExpandedIds;
 
+  // need to filter the expandedIds to only include items that are present in the tree
+  // we should not include parents that are above the current player root
+  const availableItemIds = itemsToShow?.map(({ id: itemId }) => itemId);
```

Comment:
Why does `itemsToShow` contain "empty" id in the first place? Maybe you want to filter higher in the logic.

Reply:
Not sure I get your question. This line simply extracts the ids of the available items. It is not used for filtering empty items.